recipes/inja/all/conanfile.py

```diff
@@ -1,41 +1,64 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
 import os
-from conans import ConanFile, tools
 
-required_conan_version = ">=1.28.0"
+required_conan_version = ">=1.50.0"
+
 
 class InjaConan(ConanFile):
     name = "inja"
     license = "MIT"
     homepage = "https://github.com/pantor/inja"
     url = "https://github.com/conan-io/conan-center-index"
     description = "Inja is a template engine for modern C++, loosely inspired by jinja for python"
-    topics = ("conan", "jinja2", "string templates", "templates engine")
+    topics = ("jinja2", "string templates", "templates engine")
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def requirements(self):
-        self.requires("nlohmann_json/3.9.1")
+        self.requires("nlohmann_json/3.11.2")
 
     def package_id(self):
-        self.info.header_only()
+        self.info.clear()
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, 11)
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
```

Comment:
Layout without any build method 🤔 is this required (AFAIK it should not have much impact)

Reply:
It has an impact: https://github.com/conan-io/conan/issues/11867#issuecomment-1219290365

Reply:
Yicks. Okay that's for the reference.

I'll make sure layouts are added now